go/arrow/ipc/ipc.go

```diff
@@ -0,0 +1,71 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipc // import "github.com/apache/arrow/go/arrow/ipc"
+
+import (
+	"io"
+
+	"github.com/apache/arrow/go/arrow/memory"
+)
+
+const (
+	errNotArrowFile             = errString("arrow/ipc: not an Arrow file")
```

Comment:
```suggestion
	errNotArrowFile             = errors.New("arrow/ipc: not an Arrow file")
```

Reply:
the advantage of using `errString` is that `errNotArrowFile` (and others) can be `const` instead of variables.

I can switch to `errors.New` if you prefer.

Reply:
I'll assume you don't mind :)